sprout/appliances/models.py

```diff
@@ -607,6 +607,17 @@ def set_status(self, status):
             template.save()
             self.logger.info("{}: {}".format(self.pk, status))
 
+    @property
+    def possibly_bunk(self):
```

Comment:
**REQUIRED:** Can we have a docstring for this weirdly named method

Reply:
Sure